public/components/FormControls/FormikFieldNumber/FormikFieldNumber.js

```diff
@@ -42,12 +42,13 @@ const FormikFieldNumber = ({
   />
 );
 
-const FieldNumber = ({ name, form, field, inputProps: { onChange, isInvalid, ...rest } }) => (
+const FieldNumber = ({ name, form, field, inputProps: { onChange, isInvalid, disabled, ...rest } }) => (
   <EuiFieldNumber
     {...field}
     {...rest}
     onChange={e => (typeof onChange === 'function' ? onChange(e, field, form) : field.onChange(e))}
     isInvalid={typeof isInvalid === 'function' ? isInvalid(name, form) : isInvalid}
+    disabled={ disabled? "disabled":"" }
```

Comment:
Can't we leverage `...rest` here and let consumer decide what to pass on ?

Reply:
Thanks, will change it.